cmd/juju/commands/bootstrap.go

```diff
@@ -75,6 +75,10 @@ algorithm by assigning a "placement directive" via the '--to' option. This
 dictates what machine to use for the controller. This would typically be
 used with the MAAS provider ('--to <host>.maas').
 
+Available keys for use with --config can be found here:
+    https://jujucharms.com/docs/2.2/controllers-config
```

Comment:
https://jujucharms.com/docs/stable/controllers-config

^^^^ use the stable URL, here and below etc